Jenkinsfile_CNP

```diff
@@ -6,16 +6,10 @@ String product = "cmc"
 String component = "citizen-frontend"
 
 List<LinkedHashMap<String, Object>> secrets = [
+  //for smoke tests
```

Comment:
not sure if this is needed but its smoke / functional tests

Reply:
I believe it is for the docker images that are run up for smoke/functional.